examples/js/loaders/VRMLLoader.js

```diff
@@ -3331,7 +3336,7 @@ THREE.VRMLLoader = ( function () {
 
 			$.AT_LEAST_ONE( function () {
 
-				$.OR( [
+				$.OR( $.altSingleFieldValue || ($.altSingleFieldValue = [
```

Comment:
This change is related to [Caching Arrays of Alternatives](https://sap.github.io/chevrotain/docs/guide/performance.html#caching-arrays-of-alternatives), right?

Instead of `$.altSingleFieldValue` and `$.altMultiFieldValue`, I would prefer to just name them similar to the guide: `$.c1` and `$.c2`. This should make it more clear that `$.cX` represents cached rules